// @clerk/shared/dist/react/index.js
Object.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });
const require_createContextAndHook = require('./hooks/createContextAndHook.js');
const require_contexts = require('./contexts.js');
const require_clerk_query_client = require('./query/clerk-query-client.js');
const require_useAPIKeys = require('./hooks/useAPIKeys.js');
const require_useUserBase = require('./hooks/base/useUserBase.js');
const require_useOAuthConsent = require('./hooks/useOAuthConsent.js');
const require_useOrganizationBase = require('./hooks/base/useOrganizationBase.js');
const require_useSessionBase = require('./hooks/base/useSessionBase.js');
const require_useClerk = require('./hooks/useClerk.js');
const require_useAttemptToEnableOrganizations = require('./hooks/useAttemptToEnableOrganizations.js');
const require_useOrganization = require('./hooks/useOrganization.js');
const require_useOrganizationCreationDefaults = require('./hooks/useOrganizationCreationDefaults.js');
const require_useOrganizationList = require('./hooks/useOrganizationList.js');
const require_useSafeLayoutEffect = require('./hooks/useSafeLayoutEffect.js');
const require_useSession = require('./hooks/useSession.js');
const require_useClientBase = require('./hooks/base/useClientBase.js');
const require_useSessionList = require('./hooks/useSessionList.js');
const require_useUser = require('./hooks/useUser.js');
const require_useDeepEqualMemo = require('./hooks/useDeepEqualMemo.js');
const require_useReverification = require('./hooks/useReverification.js');
const require_useStatements = require('./hooks/useStatements.js');
const require_usePaymentAttempts = require('./hooks/usePaymentAttempts.js');
const require_usePaymentMethods = require('./hooks/usePaymentMethods.js');
const require_usePlans = require('./hooks/usePlans.js');
const require_useSubscription = require('./hooks/useSubscription.js');
const require_useCheckout = require('./hooks/useCheckout.js');
const require_useCreditBalance = require('./hooks/useCreditBalance.js');
const require_useCreditHistory = require('./hooks/useCreditHistory.js');
const require_useStatementQuery = require('./hooks/useStatementQuery.js');
const require_usePlanDetailsQuery = require('./hooks/usePlanDetailsQuery.js');
const require_usePaymentAttemptQuery = require('./hooks/usePaymentAttemptQuery.js');
const require_useUserEnterpriseConnections = require('./hooks/useUserEnterpriseConnections.js');
const require_useOrganizationEnterpriseConnections = require('./hooks/useOrganizationEnterpriseConnections.js');
const require_useOrganizationDomains = require('./hooks/useOrganizationDomains.js');
const require_useOrganizationEnterpriseConnectionTestRuns = require('./hooks/useOrganizationEnterpriseConnectionTestRuns.js');
require('./hooks/index.js');
const require_ClerkContextProvider = require('./ClerkContextProvider.js');
const require_payment_element = require('./billing/payment-element.js');
const require_PortalProvider = require('./PortalProvider.js');

exports.ClerkContextProvider = require_ClerkContextProvider.ClerkContextProvider;
exports.ClerkInstanceContext = require_contexts.ClerkInstanceContext;
exports.InitialStateProvider = require_contexts.InitialStateProvider;
exports.OptionsContext = require_contexts.OptionsContext;
exports.UNSAFE_PortalProvider = require_PortalProvider.UNSAFE_PortalProvider;
exports.__createClerkTestQueryClient = require_clerk_query_client.__createClerkTestQueryClient;
exports.__experimental_CheckoutProvider = require_contexts.__experimental_CheckoutProvider;
exports.__experimental_PaymentElement = require_payment_element.PaymentElement;
exports.__experimental_PaymentElementProvider = require_payment_element.PaymentElementProvider;
exports.__experimental_useCheckout = require_useCheckout.useCheckout;
exports.__experimental_usePaymentAttempts = require_usePaymentAttempts.usePaymentAttempts;
exports.__experimental_usePaymentElement = require_payment_element.usePaymentElement;
exports.__experimental_usePaymentMethods = require_usePaymentMethods.usePaymentMethods;
exports.__experimental_usePlans = require_usePlans.usePlans;
exports.__experimental_useStatements = require_useStatements.useStatements;
exports.__experimental_useSubscription = require_useSubscription.useSubscription;
exports.__internal_useClientBase = require_useClientBase.useClientBase;
exports.__internal_useCreditBalanceQuery = require_useCreditBalance.__internal_useCreditBalanceQuery;
exports.__internal_useCreditHistoryQuery = require_useCreditHistory.__internal_useCreditHistoryQuery;
exports.__internal_useOrganizationBase = require_useOrganizationBase.useOrganizationBase;
exports.__internal_useOrganizationDomains = require_useOrganizationDomains.useOrganizationDomains;
exports.__internal_useOrganizationEnterpriseConnectionTestRuns = require_useOrganizationEnterpriseConnectionTestRuns.useOrganizationEnterpriseConnectionTestRuns;
exports.__internal_useOrganizationEnterpriseConnections = require_useOrganizationEnterpriseConnections.useOrganizationEnterpriseConnections;
exports.__internal_usePaymentAttemptQuery = require_usePaymentAttemptQuery.usePaymentAttemptQuery;
exports.__internal_usePlanDetailsQuery = require_usePlanDetailsQuery.__internal_usePlanDetailsQuery;
exports.__internal_useSessionBase = require_useSessionBase.useSessionBase;
exports.__internal_useStatementQuery = require_useStatementQuery.useStatementQuery;
exports.__internal_useUserBase = require_useUserBase.useUserBase;
exports.__internal_useUserEnterpriseConnections = require_useUserEnterpriseConnections.useUserEnterpriseConnections;
exports.__resetClerkQueryClientForTest = require_clerk_query_client.__resetClerkQueryClientForTest;
exports.__setClerkQueryClientForTest = require_clerk_query_client.__setClerkQueryClientForTest;
exports.assertContextExists = require_createContextAndHook.assertContextExists;
exports.createContextAndHook = require_createContextAndHook.createContextAndHook;
exports.getClerkQueryClient = require_clerk_query_client.getClerkQueryClient;
exports.isDeeplyEqual = require_useDeepEqualMemo.isDeeplyEqual;
exports.useAPIKeys = require_useAPIKeys.useAPIKeys;
exports.useAssertWrappedByClerkProvider = require_contexts.useAssertWrappedByClerkProvider;
exports.useAttemptToEnableOrganizations = require_useAttemptToEnableOrganizations.useAttemptToEnableOrganizations;
exports.useClerk = require_useClerk.useClerk;
exports.useClerkInstanceContext = require_contexts.useClerkInstanceContext;
exports.useDeepEqualMemo = require_useDeepEqualMemo.useDeepEqualMemo;
exports.useInitialStateContext = require_contexts.useInitialStateContext;
exports.useOAuthConsent = require_useOAuthConsent.useOAuthConsent;
exports.useOptionsContext = require_contexts.useOptionsContext;
exports.useOrganization = require_useOrganization.useOrganization;
exports.useOrganizationCreationDefaults = require_useOrganizationCreationDefaults.useOrganizationCreationDefaults;
exports.useOrganizationList = require_useOrganizationList.useOrganizationList;
exports.usePortalRoot = require_PortalProvider.usePortalRoot;
exports.useReverification = require_useReverification.useReverification;
exports.useSafeLayoutEffect = require_useSafeLayoutEffect.useSafeLayoutEffect;
exports.useSession = require_useSession.useSession;
exports.useSessionList = require_useSessionList.useSessionList;
exports.useUser = require_useUser.useUser;
